apps/prairielearn/src/lib/question.js

```diff
@@ -1522,6 +1577,22 @@ module.exports = {
   getAndRenderVariant(variant_id, variant_seed, locals, callback) {
     async.series(
       [
+        (callback) => {
+          if (locals.question.course_id === locals.course.id) {
+            locals.question_course = locals.course;
+            callback(null);
+          } else {
+            sqldb.queryOneRow(
+              sql.select_question_course,
+              { question_course_id: locals.question.course_id },
+              (err, result) => {
+                if (ERR(err, callback)) return;
+                locals.question_course = result.rows[0].course;
+                callback(null);
+              }
+            );
+          }
+        },
```

Comment:
Ditto re: `async` function. Having seen this three times, maybe this should be a little reusable function?